app/jobs/mentions/create_all_job.rb

```diff
@@ -0,0 +1,10 @@
+module Mentions
+  class CreateAllJob < ApplicationJob
+    queue_as :mentions_create_all
+
+    def perform(notifiable_id, notifiable_type)
+      notifiable = notifiable_type.constantize.find_by(id: notifiable_id)
```

Comment:
Since `constantize` is used, it's safer to check what class names are allowed here, e.g. return if another `notifiable_type` is passed